doc/user_guide/user_guide.md

```diff
@@ -109,13 +109,30 @@ Transformers Extension Package. See [the latest release](https://github.com/exas
       --language-alias <LANGUAGE_ALIAS> \ 
       --version <RELEASE_VERSION> \
       --ssl-cert-path <ssl-cert-path> \
-      --use-ssl-cert-validation
+      --use-ssl-cert-validation \
```

Comment:
Is here after something missing?

Reply:
Good call. I started adding something, then abandoned it.